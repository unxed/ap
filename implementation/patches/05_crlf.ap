version: "1.0"
changes:
  - file_path: "05_crlf.txt"
    newline: CRLF
    modifications:
      - action: REPLACE
        target:
          snippet: "Second line to be replaced."
        content: "This is the new second line."