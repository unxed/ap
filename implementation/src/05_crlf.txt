First line.
Second line to be replaced.
Third line.