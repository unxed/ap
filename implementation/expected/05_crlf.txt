First line.
This is the new second line.
Third line.